neo/Wallets/NEP5Token.py

```diff
@@ -130,7 +230,20 @@ def Approve(self, wallet, owner_addr, requestor_addr, amount):
         return tx, fee, results
 
     def Mint(self, wallet, mint_to_addr, attachment_args):
-
+        """
+        Call the "mintTokens" function of the smart contract.
+
+        Args:
+            wallet (neo.Wallets.Wallet): a wallet instance.
+            mint_to_addr (str): public address of the account to mint the tokens to.
+            attachment_args: ???
```

Comment:
@localhuman you probably know what this parameter type + description should be.

Reply:
should be `(str): passed from prompt, should be '--attach-neo=X' or '--attach-gas=X'`